deploy/olm-catalog/integreatly-operator/integreatly-operator-1.18.0/integreatly-operator.v1.18.0.clusterserviceversion.yaml

```diff
@@ -6,55 +6,54 @@ metadata:
       [
         {
           "apiVersion": "integreatly.org/v1alpha1",
-          "kind": "Installation",
+          "kind": "RHMI",
           "metadata": {
-            "name": "example-installation"
+            "name": "INSTALLATION_NAME"
           },
           "spec": {
-            "namespacePrefix": "redhat-rhmi-",
-            "selfSignedCerts": true,
-            "smtpSecret": "redhat-rhmi-smtp",
-            "type": "managed",
-            "useClusterStorage": true
+            "namespacePrefix": "INSTALLATION_PREFIX-",
+            "operatorsInProductNamespace": "OPERATORS_IN_PRODUCT_NAMESPACE",
+            "selfSignedCerts": "SELF_SIGNED_CERTS",
+            "smtpSecret": "INSTALLATION_PREFIX-smtp",
+            "type": "INSTALLATION_TYPE",
+            "useClusterStorage": "USE_CLUSTER_STORAGE"
           }
         },
         {
           "apiVersion": "integreatly.org/v1alpha1",
-          "kind": "Installation",
+          "kind": "RHMI",
           "metadata": {
-            "name": "INSTALLATION_NAME"
+            "name": "example-rhmi"
           },
           "spec": {
-            "namespacePrefix": "INSTALLATION_PREFIX-",
-            "selfSignedCerts": "SELF_SIGNED_CERTS",
-            "smtpSecret": "INSTALLATION_PREFIX-smtp",
-            "type": "INSTALLATION_TYPE",
-            "useClusterStorage": "USE_CLUSTER_STORAGE"
+            "namespacePrefix": "redhat-rhmi-",
+            "selfSignedCerts": true,
+            "smtpSecret": "redhat-rhmi-smtp",
+            "type": "managed",
+            "useClusterStorage": true
           }
         }
       ]
     capabilities: Basic Install
     categories: Integreating greatly
     certified: "false"
-    containerImage: quay.io/integreatly/integreatly-operator:v1.17.0
+    containerImage: quay.io/integreatly/integreatly-operator:v1.18.0
     createdAt: "2019-10-15 10:39:00"
     description: Greatly integrating integreatly greatness
     support: Integreatly
-  name: integreatly-operator.v1.17.0
+  name: integreatly-operator.v1.18.0
   namespace: placeholder
 spec:
   apiservicedefinitions: {}
   customresourcedefinitions:
     owned:
-    - description: Represents an installation of Integreatly.
-      displayName: Integreatly Installation
-      kind: Installation
-      name: installations.integreatly.org
-      specDescriptors:
-      - description: The desired integreatly products.
-        displayName: Products
-        path: products
```

Comment:
Just testing an olm install via `make test/e2e/olm`. Looks like these 2 fields are required for `specDescriptors` for a successfull install (subscription won't resolve without it)